### PR TITLE
chore: version update

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.

AB#{1931}-https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%205?workitem=1931

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/ssr/package-lock.json [hunk 1]
```diff
@@ -14838,66 +15285,74 @@
...
     "next": {
-      "version": "9.3.5",
-      "resolved": "https://registry.npmjs.org/next/-/next-9.3.5.tgz",
-      "integrity": "sha512-vJKGUK2nMsveYZTZH5Oz9L+cDYcCsuWLUXfWJejvhdQ3jLgVhZVa+7TQ45Kt0PA1ea7jqlDUN22DtzmPFi981w==",
-      "requires": {
-        "@ampproject/toolbox-optimizer": "2.2.0",
-        "@babel/core": "7.7.2",
-        "@babel/plugin-proposal-class-properties": "7.7.0",
-        "@babel/plugin-proposal-nullish-coalescing-operator": "7.7.4",
+      "version": "9.4.1",
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/package-lock.json:15288]: Do we need to pin this version?

File: packages/scaffolding-cli/templates/src/ssr/package-lock.json [hunk 2]
```diff
@@ -19901,9 +20575,9 @@
       }
     },
     "typescript": {
-      "version": "3.8.3",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.8.3.tgz",
-      "integrity": "sha512-MYlEfn5VrLNsgudQTVJeNaQFUAI7DkhnOjdpAp4T+ku1TfQClewlbSuTVHiA+8skNBgaf02TL/kLOvig4y3G8w==",
+      "version": "3.9.3",
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/package-lock.json:20578]: Can we remove the the pinned version from README and package.json?

Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/package-lock.json:20578]: Updated the readmes

File: packages/scaffolding-cli/templates/src/ssr/package-lock.json (hunk 1 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/package-lock.json:15288]: I reckon we leave with ~9.4 as this is a new release. we will expect some patches. What do you say? @estherthetesteramido

**Pull request merged**